### DISCUSSION TITLE
Why are some images not shown on my website?

### DISCUSSION THREAD

**Discussion by nibreh** [datenstrom/community#175] [Ask a question] [answered]
I'v got an image named adlib.jpg

`[image adlib.jpg adlib]` doesn't show the image

Tested with image named `ad.jpg` and doesn't work

but strangely, `adc.jpg` is working

Perhaps an explanation ?

Tested on a fresh install with yellow-master

**Comment by markseuffert** [comment, marked as answer]
It's probably blocked in the web browser by AdBlock, AdBlock Plus or similar.

**Comment by nibreh** [comment]
That's right !

I'm using Ublock Origin and Privacy Badger with Firefox.
I Always forget to deactivate them on certains sites.
Hope this issue can be helpful for some users.

Thanks
